aggsender/query/ler_query.go

```diff
@@ -52,3 +50,17 @@ func (l *lerDataQuerier) GetLastLocalExitRoot() (common.Hash, error) {
 
 	return rollupData.LastLocalExitRoot, nil
 }
+
+// GetStartLER retrieves the starting Local Exit Root (LER) for the rollup
+func (l *lerDataQuerier) GetStartLER() (common.Hash, error) {
+	ler, err := l.GetLastLocalExitRoot()
+	if err != nil {
+		return common.Hash{}, fmt.Errorf("failed to get last local exit root: %w", err)
+	}
+
+	if ler == aggkitcommon.ZeroHash {
+		return types.EmptyLER, nil
+	}
+
+	return ler, nil
+}
```

Comment:
Why don't we merge this function into `GetLastLocalExitRoot` (https://github.com/agglayer/aggkit/blob/416c39c080e8e3df9b1511430c038ffc97a6e545/aggsender/query/ler_query.go#L45-L52)? Especially because I find the `GetStartLER` name misleading (it is more convenient to name it `GetLastLocalExitRoot`, and if we merge the logic into that one it is a win win situation IMO).

Reply:
Fix: https://github.com/agglayer/aggkit/pull/846/commits/95a6b77ae12cb374017c4a58f4ca51f7fa1f8b5c